### DISCUSSION TITLE
Are there any file extensions that won't be encrypted by Kryptor?

### DISCUSSION THREAD

**Discussion by inferno-prime** [samuel-lucas6/Kryptor#22] [Q&A] [answered]
I have a folder with some different filetypes inside and not all of them seem to be encrypting. Are there any file extensions that wont be encrypted by kryptor ?

**Comment by samuel-lucas6** [comment, marked as answer]
Hi @inferno-prime,

Thanks for reporting an issue. Kryptor should encrypt all file types apart from '.kryptor' files. As a work around, you should be able to encrypt these files by selecting them by holding down Ctrl and clicking before dragging them onto the Kryptor window or by using File => Select Files. 

Please can you give me some more details about the problem such as the file types, the version of Kryptor, whether this is a parent folder or a subfolder, whether you have the 'Overwrite Files' setting enabled, how many files were encrypted, how many files should have been encrypted, etc. Please can you also upload/attach your error log found at either ('%APPDATA%\Kryptor\error log.txt' if installed or 'Kryptor/error log.txt' if you're using a portable version). This error log doesn't contain any personal/sensitive information, just details about any errors you've had. Feel free to upload a screenshot as well if you'd like to.

**Comment by inferno-prime** [comment]
I did not have overwrite files enabled apologies for wasting your time.

**Comment by samuel-lucas6** [comment]
> I did not have overwrite files enabled apologies for wasting your time.

No worries! I'm glad you've sorted the problem.
